### PR TITLE
Added rotation along the axis in Solar System Animations example

### PR DESCRIPTION
**Pull request by xtanion**
This PR improves the [Solar System Animation](https://github.com/fury-gl/fury/blob/master/docs/tutorials/01_introductory/viz_solar_system.py) example by adding rotation along the axis of each planets & changing revolution of planets to counter-clockwise direction.
Here's a preview: 

https://user-images.githubusercontent.com/74976752/148289559-f7579692-42dd-42e2-bb5b-096d2d9e0ba9.mp4

### CODE REVIEW

**Review by skoudoro — COMMENTED** [review]
Thank you for this @xtanion,

We will look into it. Can you fix all the pep8 issues.
Thanks.

**Comment by xtanion** [comment]
@skoudoro I've fixed all PEP8 issues, Please have a look. 
Thanks in Advance

**Review by skoudoro — COMMENTED** [review]
Hi @xtanion,

Can you address my comment below and then I can merge this PR.

Venus seems to rotate really slow, it is hard to see. Despite that, evrything seems to work well! Thank you

File: docs/tutorials/01_introductory/viz_solar_system.py [hunk 1]
```diff
@@ -70,6 +70,8 @@ def init_planet(planet_data):
     planet_image = io.load_image(planet_file)
     planet_actor = actor.texture_on_sphere(planet_image)
     planet_actor.SetPosition(planet_data['position'], 0, 0)
+    if(planet_data['filename'] != '8k_saturn_ring_alpha.png'):
```
[the remark this answers is not in the document]
Comment (xtanion) [reply docs/tutorials/01_introductory/viz_solar_system.py]: Hey, @skoudoro I removed the parenthesis. Venus makes a rotation about its axis in about 243 earth days, while other planets are significantly faster.

**Comment by skoudoro** [comment]
Thank you @xtanion

**Pull request merged**